defaults/main.yml

```diff
@@ -1,16 +1,4 @@
 ---
-# The LOCAL directory where the WireGuard certificates are stored after they
-# were generated. By default this will expand to user's LOCAL ${HOME}
-# (the user that run's "ansible-playbook" command) plus
-# "/wireguard/certs". That means if the user's ${HOME} directory is e.g.
-# "/home/da_user" then "wireguard_cert_directory" will have a value of
-# "/home/da_user/wireguard/certs". If you change this make sure that
-# the parent directory is writable by the user that runs "ansible-playbook"
-# command.
-wireguard_cert_directory: "{{ '~/wireguard/certs' | expanduser }}"
-wireguard_cert_owner: "root"
-wireguard_cert_group: "root"
-
```

Comment:
This part also needs to be removed from README.md in the `Role Variables` section.

Reply:
Also the CHANGELOG.md needs to be updates. I guess when we ready with this PR we need to adjust it a little bit again but for now I guess we can already add this text:

```
**4.0.0**

- IMPORTANT: Version 4.0.0 has some breaking changes and is no longer compatible with versions < 4.0.0!
- Role is now stateless. It doesn't store private and public keys anymore on your local machine. Therefore the variables `wireguard_cert_directory`, `wireguard_cert_owner` and `wireguard_cert_group` are now obsolete and no longer needed (contribution by pierreozoux)
```
If you merge the code for supporting multiple Wireguard interfaces into this PR then the CHANGELOG text above needs to be extended e.g.:
```
- Role now supports multiple Wireguard interfaces. README was updated accordingly (contribution by pierreozoux)
```
So please also update README accordingly when you implement https://github.com/githubixx/ansible-role-wireguard/issues/25.